examples/http_routes_oak_crud_middleware_with_sqlite3_db.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * @title HTTP server: Performing CRUD operations using SQLite3
+ * @difficulty intermediate
+ * @tags web, cli, deploy
+ * @run -A <url>
+ * @group Network
+ *
+ * An example of a HTTP server for CRUD routes with oak middleware framework and SQLite3 database.
+ * It demonstrates the CRUD(Create, Read, Update and Delete) operations on file-based SQLite Database using HTTP methods (Get, Post, Put, Delete, Options)
+ */
+
+import { Application, Router } from "https://deno.land/x/oak@v11.1.0/mod.ts";
```

Comment:
you can simply use `jsr:@oak/oak` here, unless you're are doing this specifically to use v11.1.0.